autoload/maktaba/syscall.vim

```diff
@@ -216,6 +284,31 @@ endfunction
 
 ""
 " @dict Syscall
+" Executes the system asynchronously and invokes the callback on completion.
```

Comment:
Could you document the `v:servername` and clientserver conditions that determine whether it's available? There are some big tradeoffs, basically if you want broad support for console vim without users having to think about it, you need allow_sync_fallback and consider warning users, so I want to make sure plugin authors have all the info to make an informed decision.